ui/lib/core/addon/components/form-field.hbs

```diff
@@ -16,22 +16,41 @@
   {{/unless}}
   {{#if @attr.options.possibleValues}}
     {{#if (eq @attr.options.editType "radio")}}
-      <div class="control columns" data-test-radio-input>
+      <div class="control {{unless this.isRadioGroup 'columns'}}" data-test-radio-input>
         {{#each (path-or-array @attr.options.possibleValues @model) as |val|}}
-          <div class="column is-narrow is-flex-center has-text-grey has-right-margin-s" data-test-input={{@attr.name}}>
+          <div
+            class="is-flex-center
+              {{unless
+                this.isRadioGroup
+                'column is-narrow has-text-grey has-right-margin-s'
+                'has-top-padding-xs has-bottom-padding-s'
+              }}"
+            data-test-input={{@attr.name}}
+          >
             <RadioButton
               class="radio"
               name={{@attr.name}}
               id={{or val.value val}}
               value={{or val.value val}}
-              @value={{or val.value val}}
+              @value={{if (or (eq @attr.type "boolean") val.value) val.value val}}
```

Comment:
It's tricky to manage or's with boolean values in the template. I think rather than having the possibleValues be booleans, maybe we take a note from the [LDAP libraries](https://github.com/hashicorp/vault/blob/e24301e32d1ab81c7e035a5d9b918136f46c2d6b/ui/app/models/ldap/library.js#L56C4-L56C4) book and use strings that are transformed to boolean in the serializer. Alternatively, you could do the same thing but define it as a [custom transform](https://github.com/hashicorp/vault/blob/main/ui/app/transforms/mount-visibility.js) like mount-config's `listVisibility`

Reply:
I paired with Claire and followed something similar to what's done in [secrets sync](https://github.com/hashicorp/vault/pull/24413/files#diff-66a4d087a5b1710a98cab7db466d43a5aff94ec9c47eb6bb8720ca02a6a4a52bR50-R69). Instead, we can use a getter/setter and utilize the `fieldValue` to get and set the correct values for the forms. The forms expect the strings, but the api and query params are currently set up for them being booleans. By doing it this way, we can keep the attr type from the api and also maintain what's needed for the forms.